File: 4070.py
T = int(input())

for test_case in range(1, T + 1):
    N = int(input())

    dp = [1 for _ in range(N + 1)]

    for i in range(2, N + 1):
        dp[i] = dp[i - 1] + dp[i - 2] * 2

    print(f'#{test_case} {dp[N]}')

File: 4070.cpp
#include <iostream>

using namespace std;

unsigned long long dp[26];

int main(int argc, char **argv)
{
    ios::sync_with_stdio(0);
    cin.tie(nullptr);

    int test_case;
    int T;

    cin >> T;

    for (test_case = 1; test_case <= T; ++test_case)
    {
        int N;

        cin >> N;

        dp[0] = 1;
        dp[1] = 1;

        for (int i = 2; i <= N; i++)
            dp[i] = dp[i - 1] + dp[i - 2] * 2;

        cout << '#' << test_case << ' ' << dp[N] << '\n';
    }
    return 0; // 정상종료시 반드시 0을 리턴해야합니다.
}